tests/centos8/install.sh

```diff
@@ -52,24 +49,23 @@ init() {
 }
 
 case "${1}" in
-    repo)
-        prepare_local_yum "${2}"
-        ;;
-    install)
-        install
-        init
-        ;;
-    "")
+repo)
+    prepare_local_yum "${2}"
+    ;;
+install)
+    install
+    init
+    ;;
+"")
+    prepare_local_yum
+    install
+    init
+    ;;
+*)
+    if [ -d "${1}" ]; then
         prepare_local_yum
-        install
-        init
-        ;;
-    *)
-        if [ -d "${1}" ]; then
-            prepare_local_yum
-        fi
-        install
-        init
-        ;;
+    fi
```

Comment:
Most of this is lift-shift from the old stuff I actually plan to trim these down to just what we need.
The diff here is just because my editor reformatted everything.